# 0xF4 : Status Message - Port Information

# Variables declaration
	uint8 SyncByte  # IMU Address
	uint8 MessageID  # Message ID
	uint32 baud_rate  # Baud Rate
	uint32 bytes_transmitted  # Bytes transmitted since boot
	uint16 Checksum  # Uint16 Checksum. Complement = False
